RunningTheProduct.md

```diff
@@ -1,45 +1,44 @@
 ### Running the Product
 
+# Run the program by running the RedistricitingDriver.java file.
 
-##Run the program by running the RedistricitingDriver.java file.
 
-
-#NOTE: COLUMNS * ROWS MUST BE A PERFECT SQUARE
+*NOTE: COLUMNS * ROWS MUST BE A PERFECT SQUARE*
 
 1. First enter the number of rows
-the default number of rows is 5
-
-#Then enter the number of columns
-The default number of columns is 5
-
-#Lastly the number of districts to be created
-
-##Sample Input/Output :
-Please enter the number of rows 1
-Please enter the number of columns 16
-Please enter the number of districts 8
-Size of region: 1 x 16
-Number of districts: 8
-Created region swdmt.redistricting.Region@37a71e93 with 16 locations
-+---+---+---+---+
-| * | * | * | * |
-+---+---+---+---+
-| * | * | * | * |
-+---+---+---+---+
-| * | * | * | * |
-+---+---+---+---+
-| * | * | * | * |
-+---+---+---+---+
-
-Assigned voters with arbitrary parties to locations.
-+---+---+---+---+
-| 0 | 1 | 1 | 1 |
-+---+---+---+---+
-| 0 | 1 | 1 | 1 |
-+---+---+---+---+
-| 0 | 1 | 1 | 0 |
-+---+---+---+---+
-| 1 | 1 | 0 | 1 |
-+---+---+---+---+
-Created set of districts [[District@410424423; size: 2], [District@1030870354; size: 2], [District@1963387170; size: 2], [District@237061348; size: 2], [District@1190035432; size: 2], [District@1685538367; size: 2], [District@485815673; size: 2], [District@1681433494; size: 2]]
+    * the default number of rows is 5
+
+2. Then enter the number of columns
+    * The default number of columns is 5
+
+3. Lastly the number of districts to be created
+
+# Sample Input/Output :
+<p>Please enter the number of rows 1 <br>

```

Comment:
To show a transcript, preface the whole section with  \`\`\` and end it with \`\`\`

Here's a sample of that appearance:
```
Something that looks like it came from a terminal.
A prompt asking for the user to enter a number: 42
The user entered 42.
That's all!
```

Thanks!